src/amo/components/Home/index.js

```diff
@@ -266,7 +270,9 @@ export class HomeBase extends React.Component {
                 featured: true,
               },
             }}
+            header={i18n.gettext('Featured themes')}
             loading={loading}
+            placeholderCount={LANDING_PAGE_THEME_COUNT}
```

Comment:
As discussed above, if we just pass `isTheme` to `LandingAddonsCard` we could remove this logic too.

Reply:
i guess imo the placeholder and isTheme seemed to be doing the same thing so why not just pass in the placeholder prop..?

Reply:
otherwise there would be no way to override the placeholder value for a theme, right?

Reply:
It looks like you went the `isTheme` route, rather than the `placeholderCount` route, so I assume your comments above are no longer relevant? You could always pass in `placeholderCount` instead if you really needed to override the number of placeholders, right?

Reply:
yep, I did stick with the isTheme route b/c I saw as you say here, I could override this. So I think it's good now : ) thanks